src/rpdk/core/jsonutils/resolver.py

```diff
@@ -131,16 +133,18 @@ def _get_multiple_lang_type(schema_type):
         return ResolvedType(ContainerType.MULTIPLE, schema_type)
 
     @staticmethod
-    def _get_primitive_lang_type(schema_type):
-        return ResolvedType(ContainerType.PRIMITIVE, schema_type)
+    def _get_primitive_lang_type(schema_type, property_schema):
+        return ResolvedType(
+            ContainerType.PRIMITIVE, schema_type, property_schema.get("format", DEFAULT)
+        )
```

Comment:
did you test it with python/go plugin?

Reply:
it's not used by them currently so should be fine. if you look it's just adding a member variable, so other plugins should behave the same. can test to ensure though.